aws_lambda_powertools/utilities/validation/exceptions.py

```diff
@@ -1,6 +1,6 @@
 from typing import Any, List, Optional
 
-from ...exceptions import InvalidEnvelopeExpressionError
+from aws_lambda_powertools.exceptions import InvalidEnvelopeExpressionError
```

Comment:
is this correct?

Reply:
I don't think this is correct, but we shouldn't remove this. Now we don't know if customers are importing `InvalidEnvelopeExpressionError` from the validator feature.